foundation/foundation-mda/src/main/java/com/boozallen/aiops/mda/ManualActionNotificationService.java

```diff
@@ -944,7 +961,7 @@ private NotificationParams configureNotification(File optionalRoot, String profi
         params.setPomFilePath(deployDir + File.separator + "pom.xml");
         params.setProfileConfiguration("<profile>" + profile + "</profile>");
         params.setPomFile(new File(params.getPomFilePath()));
-        params.setKey(getMessageKey(params.getPomFilePath(), "execution", appName));
```

Comment:
A/Q: Feels like it would be more consistent to have the relative path to the POM file here.  Can we not relativize against the root like we have elsewhere? (Not sure why it's called "optionalRoot" given the code just uses it directly for `setPomFilePath`.)

Reply:
Update to set key with relative path as part of this function. Also update `optionalRoot` to `rootDir`.